cmd/all-in-one/Dockerfile

```diff
@@ -1,8 +1,14 @@
-ARG base_image
-ARG debug_image
+ARG TARGET=release
 
-FROM $base_image AS release
-ARG TARGETARCH=amd64
+FROM golang:1.15-alpine AS base-image-debug
```

Comment:
I see you're using golang:1.15-alpine as base image for debug, but it will still have delve missing. I feel like we reverted back changes to this Dockerfile from my PR.

Reply:
This PR here should not revert that PR.

Reply:
Sure. i added it.

We could setup multi arch build for base images next